internal/config.go

```diff
@@ -250,6 +253,31 @@ var (
 		//File
 		"WALG_FILE_PREFIX": true,
 
+		// GOLANG
+		GoMaxProcs: true,
+
+		// Web server
+		HTTPListen:       true,
+		HTTPExposePprof:  true,
+		HTTPExposeExpVar: true,
+	}
+
+	PGAllowedSettings = map[string]bool{
```

Comment:
Can we extract these db specific settings to the corresponding packages?

Reply:
I've tried to play with moving configuration to db specific files, and it looks ugly. It requires moving all constants from config, all variables to db files, add extra functions. All this increasing mess with that code. Right now this code  looks minimal, placed in one file, easy to maintain.

Reply:
Ok, then I guess its fine as it is now